src/test/regress/expected/undistribute_table.out

```diff
@@ -340,6 +340,25 @@ drop cascades to view undis_view2
 drop cascades to view another_schema.undis_view3
 CONTEXT:  SQL statement "DROP TABLE undistribute_table.view_table CASCADE"
 NOTICE:  renaming the new table to undistribute_table.view_table
+WARNING:  "view undis_view1" has dependency to "table view_table" that is not in Citus' metadata
```

Comment:
hmmm, aren't these extra warning are very confusing during `undistribute_table`? Can you think of any way to avoid/improve these messages?

Reply:
I've added a global flag to check whether we are in a function converting the table type, so we start to give NOTICE instead of WARNING messages.